pkg/metrics/exporter.go

```diff
@@ -0,0 +1,24 @@
+package metrics
+
+import (
+	"fmt"
+	"strings"
+
+	"k8s.io/klog"
+)
+
+const prometheusExporter = "prometheus"
```

Comment:
Let's change this to 

```
const (
  prometheusExporter = "prometheus"
)
```

to be consistent with other packages

Reply:
fixed